frontend/app/routes/public/apply/$id/adult-child/mailing-address.tsx

```diff
@@ -0,0 +1,415 @@
+                  maxLength={100}
+                  autoComplete="address-level2"
+                  defaultValue={defaultState.mailingAddress?.city}
+                  errorMessage={errors?.city}
+                  required
+                />
+                <InputSanitizeField
+                  id="mailing-postal-code"
+                  name="postalZipCode"
+                  className="w-full"
+                  label={isPostalCodeRequired ? t('apply-adult-child:address.address-field.postal-code') : t('apply-adult-child:address.address-field.postal-code-optional')}
+                  maxLength={100}
+                  autoComplete="postal-code"
+                  defaultValue={defaultState.mailingAddress?.postalCode}
+                  errorMessage={errors?.postalZipCode}
+                  required={isPostalCodeRequired}
+                />
+              </div>
+
+              {mailingRegions.length > 0 && (
+                <InputSelect
+                  id="mailing-province"
+                  name="provinceStateId"
+                  className="w-full sm:w-1/2"
+                  label={t('apply-adult-child:address.address-field.province')}
+                  defaultValue={defaultState.mailingAddress?.province}
+                  errorMessage={errors?.provinceStateId}
+                  options={[dummyOption, ...mailingRegions]}
+                  required
+                />
+              )}
+              <InputSelect
+                id="mailing-country"
+                name="countryId"
+                className="w-full sm:w-1/2"
+                label={t('apply-adult-child:address.address-field.country')}
+                autoComplete="country"
+                defaultValue={defaultState.mailingAddress?.country}
+                errorMessage={errors?.countryId}
+                options={countries}
+                onChange={mailingCountryChangeHandler}
+                required
+              />
+              <InputCheckbox id="sync-addresses" name="syncAddresses" value="true" checked={copyAddressChecked} onChange={checkHandler}>
+                {t('apply-adult-child:address.home-address.use-mailing-address')}
+              </InputCheckbox>
+            </div>
+          </fieldset>
+          {editMode ? (
+            <div className="flex flex-wrap items-center gap-3">
+              <Dialog open={addressDialogContent !== null} onOpenChange={onDialogOpenChangeHandler}>
+                <DialogTrigger asChild>
+                  <LoadingButton
+                    aria-expanded={undefined}
+                    variant="primary"
+                    id="save-button"
+                    type="submit"
+                    name="_action"
+                    value={FORM_ACTION.submit}
+                    loading={isSubmitting}
+                    data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Adult:Save - Mailing address click"
```

Comment:
Same comment for the rest of the `data-gc-analytics-customclick` values in this route as well as `/home-address`

```suggestion
                    data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Adult_Child:Save - Mailing address click"
```